From c34df290ad10f9015253a0a3935a336164fb6d53 Mon Sep 17 00:00:00 2001
From: sukanka <su975853527@gmail.com>
Date: Sun, 18 May 2025 11:59:20 +0800
Subject: [PATCH 5/5] fix for boost 1.88.0

now we should use boost::process::v1

2	2	src/core/debugger/debugger.cpp

diff --git a/src/core/debugger/debugger.cpp b/src/core/debugger/debugger.cpp
index e86aae846..af3f71c0d 100644
--- a/src/core/debugger/debugger.cpp
+++ b/src/core/debugger/debugger.cpp
@@ -6,7 +6,7 @@
 #include <thread>
 
 #include <boost/asio.hpp>
-#include <boost/process/async_pipe.hpp>
+#include <boost/process/v1/async_pipe.hpp>
 
 #include "common/logging/log.h"
 #include "common/polyfill_thread.h"
@@ -326,7 +326,7 @@ private:
 
     struct ConnectionState {
         boost::asio::ip::tcp::socket client_socket;
-        boost::process::async_pipe signal_pipe;
+        boost::process::v1::async_pipe signal_pipe;
 
         SignalInfo info;
         Kernel::KScopedAutoObject<Kernel::KThread> active_thread;
-- 
2.49.0

